providers/frame/utils/createCustomChainSpec.ts

```diff
@@ -0,0 +1,68 @@
+import { ss58, testUser } from "../../../crypto/mod.ts"
+
+const DEFAULT_TEST_USER_COUNT = 100_000
+const DEFAULT_TEST_USER_INITIAL_FUNDS = 1_000_000_000_000_000_000
+
+export interface CreateCustomChainSpecProps {
+  binary: string
+  chain: string
```

Comment:
Appears to be `dev` for both contracts and polkadot providers. Is this prop necessary?

Reply:
for contracts chain is `dev` but for the polkadot binary chain could be
- `polkadot-dev`
- `kusama-dev`
- `${runtime}-dev`